packages/remote-replay-launcher/src/index.ts

```diff
@@ -17,6 +18,7 @@ import { getPort } from "./url.utils";
 
 export { TunnelData } from "./types";
 
+const POLL_LOCK_INTERVAL_MS = 30_000; // 30 seconds
```

Comment:
We can make this shorter to avoid keeping the runner on, unless there is a reason for the long polling interval? perhaps:
```suggestion
const POLL_LOCK_INTERVAL_MS = 5_000; // 5 seconds
```